go.mod

```diff
@@ -1,22 +1,37 @@
 module github.com/gocraft/dbr/v2
 
-go 1.17
+go 1.21
```

Comment:
go 1.21 was needed for the clickhouse driver. Anyway 1.17 is pretty much outdated so it made sense to upgrade